code/src/components/StartGame.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { startNewGame } from 'reducers/game';
+import swal from "sweetalert";
+import styled from 'styled-components'
+
+const StartButton = styled.button`
+  display: flex;
+  background: black;
+  color: #4CFF42;
+  font-family: 'Inconsolata', monospace;
+  font-size: 25px;
+  padding: 10px;
+`
+
+export const StartGame = () => {
+  const dispatch = useDispatch();
+  const username = useSelector((store) => store.game.username);
+
+
+  const handleOnClick = () => {
+    swal({
+      content: {
+        element: "input",
+        attributes: {
+          placeholder: "What's your name, brave adventurer?",
+          type: "text",
+        },
+      },
+    }).then((value) => {
+      if (value) {
+        dispatch(startNewGame(value));
+      }
+    });
+  };
```

Comment:
Nice using swal! We have never used it before but got inspired to do so! Clean and nice code.